### PR TITLE
Translate "catch" to Ukrainian

### PR DESCRIPTION
**Pull request by YehorBoiar**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Yehor Boiar

## Language: 

- Ukrainian

## Terms defined:

- catch_exception

### CODE REVIEW

**Review by Herobread — APPROVED** [review]

**Comment by olexandr-konovalov** [comment]
Мабуть краще не "зловити" а "перехопити"?

**Comment by olexandr-konovalov** [comment]
Sorry. accidentally closed. Reopened

**Comment by YehorBoiar** [comment]
Corrected @Herobread @olexandr-konovalov please review

File: glossary.yml [hunk 1]
```diff
@@ -2504,7 +2504,15 @@
     term: ማጥመድ (ለየት ያለ)
     def: >
       አንድን ስህተት ወይም ሌላ ያልተጠበቀ ክስተት የመፍታት ኃላፊነትን መቀበል። [R](#r_language)"ለየት ያለ ነገር ከመያዝ"ይልቅ"ሁኔታን መያዝ"ይመርጣል። [Python](#python) በሌላ በኩል፣ ለየት ያሉ ሁኔታዎችን ማሳደግና መያዝን ያበረታታል ፤ በአንዳንድ ሁኔታዎች ደግሞ ይህን ማድረግ ያስፈልጋል ።
-
+  uk:
+    term: "перехопити (помилку)"
+    def: >
+      Можливість перехопити помилку або інші несподівані події та обробити інформацію про них. 
+      У R рекомендується обробляти помилки за допомогою попередньо визначених умов, тоді
+      як в інших мовах, таких як Python, помилки та повідомлення про помилки використовуються
```
Comment (olexandr-konovalov) [inline glossary.yml:2512]: помилки та повідомлення про них

**Comment by YehorBoiar** [comment]
@olexandr-konovalov @Herobread Corrected. Please review.

**Review by olexandr-konovalov — APPROVED** [review]
LGTM! @Herobread ?

**Review by Herobread — APPROVED** [review]

**Pull request merged**